### PR TITLE
Make schema registry and kafka connect optional

### PR DESCRIPTION
**Pull request by VictorKuenstler**

### CODE REVIEW

File: backend/streams_explorer/core/config.py [hunk 1]
```diff
@@ -8,17 +8,15 @@
     settings_files=["settings.yaml"],
     load_dotenv=True,
     validators=[
-        Validator("kafkaconnect.url", must_exist=True),
-        Validator(
-            "kafkaconnect.displayed_information", must_exist=True, is_type_of=list
-        ),
+        Validator("kafkaconnect.displayed_information", is_type_of=list),
```
Comment (philipp94831) [inline backend/streams_explorer/core/config.py:11]: No default here? Maybe also put it next to kafkaconnect.url?

**Review by philipp94831 — COMMENTED** [review]
What about a test for no SR configrued?

File: backend/streams_explorer/core/config.py (hunk 1 above)
Comment (VictorKuenstler) [reply backend/streams_explorer/core/config.py]: Good point. Changed it.

File: README.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Visit our introduction [blogpost](https://medium.com/bakdata/exploring-data-pipe
 
 ### Docker Compose
 
-1. Forward the ports to Kafka Connect, Schema Registry, and Prometheus. (other integrations are optional)
+1. Forward the ports to Prometheus. (Kafka Connect, Schema Registry, or other integrations are optional)
```
Comment (philipp94831) [inline README.md:49]: and instead of or?

File: backend/streams_explorer/core/services/schemaregistry.py [hunk 3]
```diff
@@ -32,6 +32,8 @@ def get_topic_value_schema(topic: str, version: int = 1) -> dict:
     @staticmethod
     def get_newest_topic_value_schema(topic: str) -> dict:
         logger.info(f"Fetch newest schema for topic {topic}")
+        if url is None:
```
Comment (philipp94831) [inline backend/streams_explorer/core/services/schemaregistry.py:35]: does it make sense to move this check before the log?

Comment (VictorKuenstler) [reply backend/streams_explorer/core/services/schemaregistry.py:34]: Absolutely 👍

**Review by philipp94831 — APPROVED** [review]

**Pull request merged**